deploy.sh

```diff
@@ -14,6 +15,18 @@ gitCommitSubject=$(git log -1 --pretty='%h %s')
 
 declare -A loggedin
 
+if [[ "$1" == "REDEPLOY" ]]; then
+  luaFiles=$(find . -type f -name '*.lua')
+else
+  luaFiles=$1
+fi
+
+if [[ "${DEV_WIKI_BASIC_AUTH}" == "" ]]; then
+  wikiBaseUrl=${lpBaseUrl}
+else
+  wikiBaseUrl=$devBaseUrl
+fi
+
 for luaFile in $luaFiles
 do
   echo "== Checking ./$luaFile =="
```

Comment:
Does this still work with the different deploy variants? Feels like we'd get `././`